ui/src/app/pages/homepage/homepage.tsx

```diff
@@ -355,8 +355,8 @@ export const Homepage = withUserProfile()(class extends React.Component<Props, S
                                     <CustomBulletList>
                                       <CustomBulletListItem bullet='→'>
                                         Create a <StyledAnchorTag href='https://support.google.com/chrome/answer/2364824'
-                                          target='_blank'>Chrome Profile</StyledAnchorTag> with your <i>All of Us</i> Researcher
-                                        Workbench Google account. This will keep your Workbench browser sessions isolated from
+                                          target='_blank'>Chrome Profile</StyledAnchorTag> with your <AoU/> Researcher
+                                        Workbench Google account. This will keep your workbench browser sessions isolated from
```

Comment:
is this issue 9?

Reply:
Issue number 18 

RW Welcome page | Grammar&Punctuation | Under tips to get your started, Workbench should be lowercase or referred to as "Researcher Workbench" (2nd sentence in first bullet)
-- | -- | --